tests/tests.helpers.ts

```diff
@@ -0,0 +1,13 @@
+import * as Browser from "detect-browser";
+
+export const itIgnore = (browsers: string[], should: string, test: any, focus?: boolean) => {
+  if (browsers.length && browsers.indexOf(Browser.name) >= 0) {
+    return;
```

Comment:
Would returning `xit(should, test)` make more sense? It would show the skipped tests in the report.

Reply:
Actually yes, I have a second PR following that makes that change when I worked on Safari.